cmd/skywire-cli/commands/hv/hvpk.go

```diff
@@ -0,0 +1,47 @@
+package clihv
+
+import (
+	"fmt"
+	"net"
+	"os"
+	"strings"
+
+	"github.com/spf13/cobra"
+
+	"github.com/skycoin/skywire-utilities/pkg/netutil"
+	"github.com/skycoin/skywire/pkg/visor"
+)
+
+func init() {
+	if os.Getenv("SKYBIAN") == "true" {
+		RootCmd.AddCommand(pkCmd)
+	}
+	localIPs, err = netutil.DefaultNetworkInterfaceIPs()
+	if err != nil {
+		logger.WithError(err).Fatalln("Could not determine network interface IP address")
+	}
+	if len(localIPs) == 0 {
+		localIPs = append(localIPs, net.ParseIP("192.168.0.1"))
+	}
+
+	pkCmd.Flags().StringVarP(&ipAddr, "ip", "i", trimStringFromDot(localIPs[0].String())+".2:7998", "ip:port to query")
```

Comment:
The port and last byte of the IP could be factored out into a constant maybe.

Reply:
it's effectively a constant right there

If I move it to a constant I would need to put it in skyenv and then import that.. was trying to avoid this for a trivial matter